### PR TITLE
Draft of challenges

### PR DESCRIPTION
**Pull request by agude**
I hear you like examples so I wrote a huge huge huge example. :-/

### CODE REVIEW

File: _chapters/06-combat-and-conflicts.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+### Challenges
+
+In normal play, the characters will occasionally roll a skill to determine the
+direction of the story. But during particularly eventful scenes where the PCs
+need to accomplish something complicated they will have to roll multiple
+skills in close succession. That's when you turn to a **Challenge**.
+
+Challenges let each member of the group use their particular skills to help
+solve the problem. The GM will set out a list of tasks that must be
```
Comment (charles-uno) [inline _chapters/06-combat-and-conflicts.md:15]: I don't love "solve the problem". Doesn't really line up with the primary use case of crushing zombies. Maybe "overcome the obstacle"? Still not great

File: _chapters/06-combat-and-conflicts.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+> her way out she sabotaged the _Angry Pig_, injured multiple crew members,
+> and accidentally left one of her books behind. Now the _Angry Pig_ is losing
+> altitude over the great desert while the escapee's trail grows colder by the
+> moment.
+>
+> The GM sets up a challenge to see how the players proceed. He comes up with
+> three tasks:
+>
+> - Decipher the convict's book (difficulty 4)
+> - Avoid the mega cacti that are just now peaking out of the clouds
```
Comment (charles-uno) [inline _chapters/06-combat-and-conflicts.md:36]: peeking

File: _chapters/06-combat-and-conflicts.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
+---
+---
+## Conflicts and Combat
+
+{% include macros.html %}
+
+### Challenges
```
Comment (charles-uno) [inline _chapters/06-combat-and-conflicts.md:7]: Not sure I get it. This seems like normal play? But the GM is laying tracks for you

File: _chapters/06-combat-and-conflicts.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+> Meanwhile Tor Byorn has been getting ready to "help" Naka decipher the book
+> by gathering _every other written work on the boat_ into a big pile. He rolled
+> `Network` (+1) to create the aspect ~~Big Pile of study material~~ with a
+> free invocation for Naka.
+> 
+> The pyromancer frantically scans the pages, but books were never her best
+> subject (+0 `Insight`). She rolls ({{dice-pzpz}}) which isn't enough, even
+> with the +2 from Tor Byorn's aspect. She looks to the GM and says "I've
+> ~~Never met a problem my magic couldn't ignite~~. I might be a pyromaniac,
+> but I'm also a trained mage. I have studied lots of arcane books before in
```
Comment (charles-uno) [inline _chapters/06-combat-and-conflicts.md:60]: double "before". 

did we eventually prefer line breaks at sentence breaks rather than at a set column?

**Review by charles-uno — CHANGES_REQUESTED** [review]
Seems like there should be two modes of play:
- Collaborative narration: unstructured order, lots of talking, occasional rolls
- Combat: go around the table, pretty much everyone rolls each turn

This feels like something in between. I guess I don't get it

**Comment by agude** [comment]
Is it worth iterating on these now? They're like a year out of date. 😄 Presumably we'll steal some parts.

**Pull request merged**